docs/fastdds/dds_layer/domain/domainParticipantListener/domainParticipantListener.rst

```diff
@@ -1,5 +1,41 @@
-.. _dds_layer_domainParticiantListener:
+.. _dds_layer_domainParticipantListener:
 
 DomainParticipantListener
 =========================
 
+DomainParticipantListener is an abstract class defining the callbacks that will be triggered
+in response to state changes on the DomainParticipant.
+By default, all these callbacks are empty and do nothing.
+The client should implement a specialization of this class overriding the callbacks
```

Comment:
The word client can have some meaning for discovery, so I thing user is less confusing.
```suggestion
The user should implement a specialization of this class overriding the callbacks
```